### PR TITLE
Iteration 2 intergration

### PR DESCRIPTION
**Pull request by Liammurray19**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 1]
```diff
@@ -10,26 +10,34 @@
...
 
-        AMP_SPEED(1125),
+        AMP_SPEED(1125, 1125, 1125),
 
-        OFF(0);
+        OFF(0, 0, 0);
 
-        private double speed;
+        private double leftSpeed;
+        private double rightSpeed;
```
Comment (Lucasszela) [inline src/main/java/frc/robot/subsystems/Shooter.java:28]: change left and right speed to top and bottom speed

File: src/main/java/frc/robot/subsystems/Led.java [hunk 2]
```diff
@@ -79,6 +79,10 @@ public Command flash() {
         return setGlobalPattern(new FlashPattern(new Color(255, 32, 82), .5));
     }
 
+    public Command BLUE() {
```
Comment (Lucasszela) [inline src/main/java/frc/robot/subsystems/Led.java]: all capital is for constant values

File: src/main/java/frc/robot/subsystems/Led.java [hunk 3]
```diff
@@ -79,6 +79,10 @@ public Command flash() {
...
+        return setGlobalPattern(new Color(255, 0, 0));
```
Comment (Lucasszela) [inline src/main/java/frc/robot/subsystems/Led.java:84]: add function to convert rgb to current format

**Review by Lucasszela — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Shooter.java (hunk 1 above)
Comment (bot190) [reply src/main/java/frc/robot/subsystems/Shooter.java:28]: I think we want to change these to A and B and then label Alpha and Beta accordingly.

**Review by msoucy — APPROVED** [review]

**Pull request merged**